lib/openstack_query/handlers/client_side_handler_datetime.py

```diff
@@ -15,16 +17,17 @@ class ClientSideHandlerDateTime(ClientSideHandler):
     Filter functions which map to QueryPresetsDateTime are defined here
     """
 
-    def __init__(self, filter_function_mappings: PresetPropMappings):
-        super().__init__(filter_function_mappings)
+    def __init__(self, _filter_function_mappings: PresetPropMappings):
```

Comment:
Nit: this shouldn't be a private var in a constructor, even if it gets mapped there ultimately